### ISSUE TITLE
Development Environment Brownout

### ISSUE THREAD

**Issue by chrisCasebeer** [kevinschaich/mintable#179]
Today I woke up to a brownout. I'm trying to see if I can't fix it.
If anyone has any ideas. Seems we can still get a limited amount of production access.

![image](https://github.com/kevinschaich/mintable/assets/4623551/0c1b6e35-2202-4462-86d9-cb1abd1c024b)
![image](https://github.com/kevinschaich/mintable/assets/4623551/977998a2-8da1-477e-bd66-65bd27ba8b0a)

What's I've tried so far. 
Got "free" production access without credit card.

Change my plaid setup to be "Development" but with a production key.
Upon attempting to add a new bank account, I still get DEVELOPMENT_ENVIRONMENT_BROWNOUT.

An immediate fix is to search "development" and change to "production".
This appears to work again. Though....changing over tokens saved under development seems beyond my knowledge. (I just deleted my old tokens, made new production tokens)

**Comment by pcarrin2** [comment]
would very much appreciate a fix for this!! I was just about to get started with mintables but waited a couple months too late, I guess.

**Comment by mrjamesghost** [comment]
Have you managed to get any further with this man? I changed every instance of development to production and broke the whole build.

**Comment by logan-life** [comment]
Yeah, I think this has broken everything for me. Was hoping it would be as easy as swapping over to the Production environment on Plaid.

Unfortunately, as an American living in the Netherlands, I need to be able to fetch transactions from EU banks. Which for Plaid means I get kicked into the Custom Production Access Onboarding flow instead of the default one. So no free and easy prod access in Plaid.

I'm awaiting the reply from Plaid, and will be interested to see if they are flexible enough to accomodate an expat hobbyist use case like this.

**Comment by tobinsouth** [comment]
Does anyone have any advice for people trying to move into production from a developer? It looks like you really need an incorporated company for this to work.
